scripts/exgfs_wave_post_pnt.sh

```diff
@@ -233,36 +212,56 @@
     DOBLL_WAV='NO'
   fi
 
+# 1.d Linking the output files
+
+  ymdh=$(${NDATE} -"${WAVHINDH}" "${PDY}${cyc}")
+  tstart="${ymdh:0:8} ${ymdh:8:2}0000"
+  N=$(( ($FHMAX_WAV_PNT - $FHMIN_WAV) * 3600 / $DTPNT_WAV + 1 ))
```

Comment:
The `$`s here shouldn't be necessary:
```suggestion
  N=$(( (FHMAX_WAV_PNT - FHMIN_WAV) * 3600 / DTPNT_WAV + 1 ))
```

Reply:
@DavidHuber-NOAA  - I know that CI tests are running, do I hold off on committing your suggestions because of that?  Do we need to commit these are restart?  Please let me know how its best to proceed.  

@AliS-Noaa  for your awareness for the future, some minor comments from @DavidHuber-NOAA

Reply:
I'm OK with holding off on these.  When testing is complete, these can be addressed and I will rocotoboot the wave_post_pnt jobs on Hera.

Reply:
After conversation with @KateFriedman-NOAA  since a rebuild is needed for WCOSS2 testing, I've gotten the go ahead to merge this and sync with develop.  Doing that now.